### PR TITLE
update t and c wording

### PR DESCRIPTION
**Pull request by jkerr321**
Agreed with Aislinn and Lucinda that we don't need to vary payment term t+c wording based on package so updating to one agreed set of copy.

### CODE REVIEW

File: partials/payment-term-selector.html [hunk 1]
```diff
@@ -2,14 +2,15 @@
 	{{#each buttonOptions}}
 	<input type="radio" name="paymentTermSelector" value="{{this.name}}" class="o-forms__radio o-forms__radio--right ncf__payment-term-select__input o-forms__radio-button--my-custom-theme" id="{{this.name}}" {{#if this.selected}} checked="checked" {{/if}}>
 	<label for="{{this.name}}" class="o-forms__label ncf__payment-term-select__label">
-		<strong class="ncf__payment-term-select__label--title">{{this.title}}</strong>
+		<strong class="ncf__payment-term-select__label--title">{{this.displayName}}</strong>
```
[the remark this answers is not in the document]
Comment (jkerr321) [reply partials/payment-term-selector.html]: There are no tests. obvs.

[the remark this answers is not in the document]
Comment (jkerr321) [reply partials/payment-term-selector.html]: Apart from that there are now! 😁

File: partials/payment-term-selector.html [hunk 2]
```diff
@@ -2,14 +2,17 @@
 	{{#each buttonOptions}}
 	<input type="radio" name="paymentTermSelector" value="{{this.name}}" class="o-forms__radio o-forms__radio--right ncf__payment-term-select__input o-forms__radio-button--my-custom-theme" id="{{this.name}}" {{#if this.selected}} checked="checked" {{/if}}>
 	<label for="{{this.name}}" class="o-forms__label ncf__payment-term-select__label">
```
[the remark this answers is not in the document]
Comment (liamkeaton) [reply partials/payment-term-selector.html]: no liam it's the id not the name

File: tests/partials/payment-term-selector.spec.js [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+	it('should only have one radio button selected', () => {
+		const $ = context.template({
+			buttonOptions:
+				[
+					{ name: 'mockName', description: 'mock', selected: true },
+					{ name: 'mockNameTwo', description: 'mockTwo' }
+				]
+		});
+
+		expect($('input[checked]').length).to.equal(1);
```
Comment (liamkeaton) [inline tests/partials/payment-term-selector.spec.js]: Should this check that the input[checked] is actually the right one?

```suggestion
		expect($('input[checked]').attr('name')).to.equal('mockName');
```

Comment (jkerr321) [reply tests/partials/payment-term-selector.spec.js]: Good shout 👍

**Pull request merged**